### ISSUE TITLE
Flicking is blocked on mobile view

### ISSUE THREAD

**Issue by Maximepzv** [naver/egjs-flicking#831]
## Description
I seem to have a problem when the browser window is small, the slider freezes and I can't scroll horizontally.
I'm running Flicking with React.

## Steps to check or reproduce
I reproduced the problem on a sandbox. Pleaqe, toggle the reactive preview and refresh the browser view :
https://codesandbox.io/s/flicking-forked-dcqy8h

https://github.com/naver/egjs-flicking/assets/26172335/b3252581-1f4a-4557-ad3c-4bd4b7920dd7

**Comment by malangfox** [comment]
Hello @Maximepzv.

It seems that we don't have access to the demo link you left.
We are welcome if you're willing to provide a demo, could you give us another link that we can see?

**Comment by Maximepzv** [comment]
Hi @malangfox,
Thanks for your help and sorry for the dead link, here is a new one 
https://codesandbox.io/s/flicking-r675kr

**Comment by pukmajster** [comment]
Same issue here, fixed by removing "overflow: auto" from the child panels... OR setting nested={true} on the Flicking component, which fixed the mobile input and allows for scrolling within the child panels.

**Comment by daybrush** [comment]
I am closing this issue because it is old.

If you have any issues, please write a new issue.
